banking-bench/src/main.rs

```diff
@@ -36,6 +36,8 @@ use {
     },
 };
 
+const TRANSFER_TRANSACTION_COST: u32 = 1470;
```

Comment:
Where did this come from?

Reply:
transfer transaction cost is static, grabbed the value from log, but added comment to explain